.github/workflows/dockerhub.yml

```diff
@@ -1,39 +1,26 @@
-name: "DockerHub Build and Push"
+name: DockerHub Build and Push
 
 on:
   push:
     branches:
       - "develop"
       - "master"
       - "release/*"
+    tags:
+      - "*"
+  workflow_dispatch:
+
 jobs:
   dockerhub-jpo-conflictmonitor:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
-      - name: Replace Docker tag
-        id: set_tag
-        run: echo "TAG=$(echo ${GITHUB_REF##*/} | sed 's/\//-/g')" >> $GITHUB_ENV
-          
-      - name: Build
-        uses: docker/build-push-action@v3
-        with:
-          push: true
-          tags: usdotjpoode/jpo-conflictmonitor:${{ env.TAG }}
-          build-args: |
-            MAVEN_GITHUB_TOKEN_NAME=${{ vars.MAVEN_GITHUB_TOKEN_NAME }}
-            MAVEN_GITHUB_TOKEN=${{ secrets.MAVEN_GITHUB_TOKEN }}
-            MAVEN_GITHUB_ORG=${{ github.repository_owner }}
-          secrets: |
-            MAVEN_GITHUB_TOKEN: ${{ secrets.MAVEN_GITHUB_TOKEN }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+    uses: usdot-jpo-ode/actions/.github/workflows/dockerhub.yml@main
+    with:
+      image: usdotjpoode/jpo-conflictmonitor

```

Comment:
question: previously we were adding environment specific tag, is that not desired functionality anymore?